libs/editor/build.gradle

```diff
@@ -68,25 +68,18 @@ dependencies {
         }
     }
 
-    // Forcing version due to https://github.com/facebook/SoLoader/issues/94
-    // To be removed with React Native 0.70+
-    implementation("com.facebook.soloader:soloader:$facebookSoloaderVersion") {
-        version {
-            strictly facebookSoloaderVersion
-        }
-    }
-
-    implementation "com.facebook.fresco:fresco:$facebookFrescoVersion"
-    implementation "com.facebook.fresco:imagepipeline-okhttp3:$facebookFrescoVersion"
-    implementation "com.facebook.fbjni:fbjni:$facebookFbjniVersion"
-    implementation "com.facebook.react:hermes-engine:$facebookReactVersion"
-    implementation "com.facebook.react:react-native:$facebookReactVersion"
+    implementation "com.facebook.react:hermes-android:$facebookReactVersion"
+    implementation "com.facebook.react:react-android:$facebookReactVersion"
 
     // This dependency will be substituted if the `local-builds.gradle` file contains
     // `localGutenbergMobilePath`. Details for this can be found in the `settings.gradle` file.
     implementation ("$rootProject.gradle.ext.gutenbergMobileBinaryPath:$rootProject.ext.gutenbergMobileVersion") {
+        // We exclude React Native modules from older and newer versions.
+        // This is needed to avoid Duplicate class errors when building using local Gutenberg Mobile path.
         exclude module: 'hermes-engine'
         exclude module: 'react-native'
```

Comment:
In theory, we don't need these modules from the previous RN version (i.e. `0.69.4`). However, I'm experiencing Duplicate class errors when building if I remove them. Seems that the fact that some of the React Native third-party libraries ([especially the forked repositories](https://github.com/wordpress-mobile/react-native-video/blob/197c644d6d1b86d0ad86831310992e71b776c9b3/android/build.gradle#L34)) are still pointing to RN 0.69.4, has to do with this issue.